module/x/gravity/keeper/governance_proposals.go

```diff
@@ -24,17 +24,20 @@ func RegisterProposalTypes() {
 	metadata := "gravity/IBCMetadata"
 	if !govtypes.IsValidProposalType(strings.TrimPrefix(metadata, prefix)) {
 		govtypes.RegisterProposalType(types.ProposalTypeIBCMetadata)
-		govtypes.RegisterProposalTypeCodec(&types.IBCMetadataProposal{}, metadata)
+		// TODO: Migration to v0.46
```

Comment:
Some TODOs here, all relating to the same deprecated function `RegisterProposalTypeCodec`.